coffeecard/CoffeeCard.WebApi/Pages/VerifyEmail.cshtml.cs

```diff
@@ -16,11 +17,11 @@ public VerifyEmail(IAccountService accountService)
 
         [BindProperty(SupportsGet = true)] public string Token { get; set; }
 
-        public IActionResult OnGet()
+        public async Task<IActionResult> OnGet()
         {
             try
             {
-                var emailVerified = _accountService.VerifyRegistration(Token);
+                var emailVerified = await _accountService.VerifyRegistration(Token);
                 if (emailVerified)
                 {
                     TempData["resultHeader"] = "Success";
```

Comment:
Comment for the `catch` block on line 32. Why do we catch the Exception supertype here? Would it not make more sense to use an `else`